packages/wpcom-checkout/src/types.ts

```diff
@@ -33,21 +33,34 @@ export interface TaxVendorInfo {
 	 */
 	country_code: string;
 
-	/**
-	 * The localized name of the tax (eg: "VAT", "GST", etc.).
-	 */
-	tax_name: string;
-
 	/**
 	 * The mailing address to display on receipts as a list of strings (each
 	 * string should be on its own line).
 	 */
 	address: string[];
 
+	/**
+	 * An object containing tax names and corresponding vendor ids that are used for the user's country
+	 *
+	 * This will deprecate the vat_id and tax_name properties
+	 * For now, those two properties will stay in place for backwards compatibility
+	 *
+	 * Property: The localized name of the tax (eg: "VAT", "GST", etc.).
```

Comment:
Minor but I'd call this the "key" rather than the "property".

Reply:
Done. 👍